packages/auth/src/api/authentication/recaptcha.ts

```diff
@@ -33,3 +33,24 @@ export async function getRecaptchaParams(auth: Auth): Promise<string> {
     ).recaptchaSiteKey || ''
   );
 }
+
+interface GetRecaptchaConfigRequest {
+  tenantId?: string;
+  clientType?: RecaptchaClientType;
+  version?: RecaptchaVersion;
+}
+
+interface GetRecaptchaConfigResponse {
+  recaptchaKey?: string;
+}
+
+export async function getRecaptchaConfig(auth: Auth,
```

Comment:
Will this be an internal function or is it meant to be exposed to developers? If internal, please prefix with `_`

Reply:
It should be an internal function while it seems those similar api wrapper under `auth/src/api/` are not prefixed with `_`? https://github.com/firebase/firebase-js-sdk/blob/50de44c2e24ce6c200a638e0a684f476549ac3cf/packages/auth/src/api/authentication/sign_up.ts#L39

Reply:
Oh good point, I forgot this was in the API section. We need to eventually go through and update those but for now yeah it's fine to leave as is :+1:

Reply:
Sounds good 👍